app/screens/people-screen/contacts/contact-transactions.tsx

```diff
@@ -2,8 +2,8 @@ import { gql } from "@apollo/client"
 import { useI18nContext } from "@app/i18n/i18n-react"
 import * as React from "react"
 import { SectionList, Text, View } from "react-native"
-import { TransactionItem } from "../../components/transaction-item"
-import { toastShow } from "../../utils/toast"
+import { TransactionItem } from "../../../components/transaction-item"
```

Comment:
```suggestion
import { TransactionItem } from "@app/components/transaction-item"
```

this would generally be better